train.py

```diff
@@ -125,7 +125,10 @@ def apply_model(data_loader, model, generator, text_pipelines, device, oom_handl
 		x, xlen, y, ylen = x.to(device, non_blocking = True), xlen.to(device, non_blocking = True), y.to(device, non_blocking = True), ylen.to(device, non_blocking = True)
 		with torch.no_grad():
 			try:
-				logits, log_probs, olen, loss = map(model(x, xlen, y = y, ylen = ylen).get, ['logits', 'log_probs', 'olen', 'loss'])
+				pred = model(x, xlen, y = y, ylen = ylen)
```

Comment:
Please use the OnnxModelWrapper in the ONNX export branch, i.e. the output from OnnxModelWrapper should be used for the allclose check

Reply:
done